GUIDE.md

```diff
@@ -0,0 +1,81 @@
+```bash
+curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
+sudo apt-get install nodejs git build-essential
+```
+- **Fedora**:
+```bash
+sudo dnf install nodejs npm git
+```
+- **Arch Linux**:
+```bash
+sudo pacman -S nodejs npm git base-devel
+```
+- **Other**: See the [installation guide for Node.JS](https://nodejs.org/en/download/package-manager/) to install Node.JS. Consult your distributions package manager to find packages for `git` and `nodejs` if the installation guide doesn't support it.
+### OS X/macOS
+You will need to install [Homebrew](https://brew.sh/) to download the necessary packages. After Homebrew is installed:
+```bash
+brew install node git
+```
+Setting up Pegaswitch
+---------------------
+1. In a terminal window, navigate to the location you want to install Pegaswitch to, and clone the Pegaswitch git repository using `git clone https://github.com/reswitched/pegaswitch.git`.
+2. Navigate into the new `pegaswitch` directory, and run `npm i`. This will install all the modules needed for Pegaswitch to function.
+3. Start Pegaswitch with `sudo node start --setuid $(id)`. Your user id can be found by running `id`, and searching the number next to your username (typically 1000, on a single-user system).
+	- If you are using Puyo Puyo Tetris or fakenews to access Pegaswitch, add `--webapplet` to this command. 
+
+Launching Pegaswitch
+--------------------
+4. Setup a connection to the Internet on your Switch, if you haven't already. Don't stay connected for too long without setting the DNS, otherwise your Switch may download an update.
+5. On your Switch, go into Settings -> Internet -> Internet Settings, pick the network your computer running Pegaswitch is connected to, and select Change Settings.
+6. Select DNS Settings and change it from Automatic to Manual. Set the Primary and Secondary DNS settings to the IP address that Pegaswitch shows in it's console.
+7.  - 1.0.0: Follow the instructions [here](http://switchbrew.org/index.php?title=Internet_Browser#WebApplet_launch_with_Tetris) to launch Pegaswitch with Puyo Puyo Tetris. If you 	don't feel like running PPT everytime you want to use pegaswitch see fakenews below.
+	- 2.0.0 or above: Go back, and select Connect to This Network. If you set it up correctly, the connection should fail with the message "Registration is required to use this network.", and you will enter Pegaswitch.
+8. Congratulations! You have successfully run Pegaswitch on your console!
+
+Optional: Setup fakenews
+------------------------
+fakenews is an alternative way of starting Pegaswitch that uses a fake news entry installed into the news applet. This method is highly recommended for Puyo Puyo Tetris users, as it provides a much more convenient way to access Pegaswitch without needing the game. This works for all versions, however, so it's up to you if you want to use this method.
```

Comment:
Even if the name starts with a lowercase letter, we should still start out sentences with capitals.